industrial_ci/src/tests/debians.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Copyright (c) 2021, Mathias Lüdtke
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Based on https://github.com/ros-industrial/industrial_ci/issues/697#issuecomment-876293987
+
+function prepare_debians() {
+  true
+}
+
+function build_and_install_debian() (
+    set -e
+    local pkg_path=$1
+    shift
+    cd "$pkg_path"
+    ici_quiet_echo bloom-generate rosdebian --ros-distro="$ROS_DISTRO" --debian-inc="ici~"
+    ici_quiet_echo dpkg-buildpackage -b -uc -us
+    mv ../*.{buildinfo,changes,deb} .
+    ici_quiet_echo ici_asroot dpkg -i ./*.deb
```

Comment:
```suggestion
    ici_quiet_echo ici_asroot apt-get install ./*.deb
```
(will resolve dependencies as well)

Reply:
> (will resolve dependencies as well)

It should must resolve dependencies!
All dependencies should already be installed.

Reply:
you should only install build dependencies to build the package, but installing the build package could bring additional run dependencies.

Reply:
> but installing the build package could bring additional run dependencies.

I just wanted to avoid pulling in the already built packages from the buildfarm.

Reply:
but then dpkg could fail and thus your CI job (I guess).

Reply:
> but then dpkg could fail and thus your CI job (I guess).

Yes, that's what I wanted to test for..

Reply:
Just for completeness, I think it could happen that colcons topological sort builds a package A before B but A needs B to get installed. So it could be that a package is not installable after it was build. You could add an install test with all packages at the end, maybe.